gitbook/src/introduction/why-using-erigon.md

```diff
@@ -0,0 +1,42 @@
+# Why using Erigon?
+
+Erigon is an efficient Ethereum implementation designed for speed, modularity, and optimization. It achieves this by utilizing technologies like staged sync, efficient state storage, and database compression.
+
+With Erigon 3, the default configuration is changing from an archive node to a **full node**. This shift enhances efficiency, accessibility, and versatility for a wider range of users, as the full node offers faster sync times and lower resource usage for everyday operations. Archive nodes—which retain full historical data—will still be available for developers and researchers who need them.
```

Comment:
`default configuration is changing` is not relevant to `Why using Erigon?` title. E2 also had FullNode mode - just was need to add 1 cli flag.